src/environment.js

```diff
@@ -229,6 +229,96 @@ const upgradeEnv = async function( env ) {
 	start( envSlug );
 };
 
+/**
+ * Upgrades WP Local Docker to v2.6.0
+ *
+ * @param {String} env Environment
+ * @return {void}
+ */
+const upgradeEnvTwoDotSix = async function( env ) {
+	if ( undefined === env || env.trim().length === 0 ) {
+		env = await envUtils.parseEnvFromCWD();
+	}
+
+	// Need to call this outside of envUtils.getPathOrError since we need the slug itself for some functions
+	if ( env === false || undefined === env || env.trim().length === 0 ) {
+		env = await envUtils.promptEnv();
+	}
+
+	let envPath = await envUtils.getPathOrError(env);
+
+	// If we got the path from the cwd, we don't have a slug yet, so get it
+	let envSlug = envUtils.envSlug( env );
+
+	let yaml = readYaml.sync( path.join( envPath, 'docker-compose.yml' ) );
+
+
+	// Create a new object for the upgrade yaml.
+	let upgraded = Object.assign( {}, yaml );
+
+	// Upgrade image.
+	let phpVersion = yaml.services.phpfpm.image.split(':').pop();
+
+	if ( '5.5' === phpVersion ) {
+		console.warn( 'Support for PHP v5.5 was removed in the latest version of WP Local Docker.' );
+		console.error( 'This environment cannot be upgraded.  No changes were made.' );
+
+		process.exit(1);
+	}
+	upgraded.services.phpfpm.image = `10up/wp-php-fpm-dev:${phpVersion}`;
```

Comment:
please lookup the php container like in create.js line 263:

```
'image': images[`php${answers.phpVersion}`],
```

Reply:
Also note there is a special case for a Linux system. See create.js line 290

Reply:
We don't have access to the `answers` in the upgrade command.  `10updocker upgrade` does not prompt.

Reply:
oh, i see what you mean, I'll adjust

Reply:
fixed in 48dbb84